src/views/home/subviews/HomeFavoritesSubview.tsx

```
@@ -1,16 +1,16 @@
 import {
   VIEW_ID_ESTIMATIONS_STOP,
   VIEW_ID_MAP,
-} from "../../../constants/ViewConstants.jsx";
+} from "../../../constants/ViewConstants";
 
-import Nav from "../../../components/Nav.jsx";
-import Error from "../../../components/Error.jsx";
-import HomeDesktop from "../../../components/home/HomeDesktop.jsx";
-import { useI18n } from "../../../contexts/I18nContext.jsx";
+import Nav from "../../../components/Nav";
+import Error from "../../../components/Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)